mediator/src/utils/patient.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { genereateFHIRPatientResource } from "./patient";
+
+test("genereateFHIRPatientResource produces the correct format", () => {
+  const chtPatient = {
+    id: "123",
+    _id: "456",
+    name: "John Doe",
+    sex: "male",
+    date_of_birth: "200 0-01-01"
```

Comment:
I had to change it to: 
```suggestion
    date_of_birth: "2000-01-01"
```